app/scaffold/scaffold.js

```diff
@@ -144,11 +144,26 @@ class Scaffold {
             const crewmemberIdsToAdd = DOM.getSelectedValues(DOM.fromElement);
 
             // if add has been implemented, reflect changes in model and view
-            if (this.addAwayTeamMembersFn(crewmemberIdsToAdd)) {
-                model.addAwayTeamMembers(crewmemberIdsToAdd);
-                DOM.moveItemsBetweenSelects(DOM.fromElement, DOM.toElement);
+            const addMembersPromise = this.addAwayTeamMembersFn(crewmemberIdsToAdd);
+            if(!addMembersPromise){
+                return;
+            }
+
+            if(crewmemberIdsToAdd.length === 0) {
+                DOM.alertBox("Please select a crew member to add");
+            }
+            else {
+                DOM.alertBox("Adding crew member(s)");
```

Comment:
Maybe add ellipses at the end of this loading message.

Reply:
Done